src/components/MoneyRequestConfirmationListFooter.tsx

```diff
@@ -931,14 +895,8 @@ function MoneyRequestConfirmationListFooter({
                     }
                 </View>
             )}
-            {primaryFields}
-            {!shouldShowAllFields && (
-                <ShowMoreButton
-                    containerStyle={[styles.mt1, styles.mb2]}
-                    onPress={toggleShouldExpandFields}
-                />
-            )}
-            <View style={[styles.mb5]}>{shouldShowAllFields && supplementaryFields}</View>
+            {fields.filter((field) => field.shouldShow).map((field) => field.item)}
```

Comment:
```suggestion
            <View style={[styles.mb5]}>{fields.filter((field) => field.shouldShow).map((field) => field.item)}</View>
```

There's a slight difference between the two, let's keep it consistent with how it was implemented before